src/pages/heyho/Login/InputId.scss

```
@@ -0,0 +1,37 @@
+.inputs {
+  input {
+    margin: 7px;
+    width: 400px;
+    font-size: 19px;
+    background-color: #f8f7f7a8;
+    height: 40px;
+    border: 1px solid rgba(230, 226, 226, 0.61);
+    border-radius: 3px;
+    padding-left: 10px;
+  }
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

- 다음과 같이 속성 위치시켜 주세요! 
```suggestion
.inputs {
  display: flex;
  flex-direction: column;
  align-items: center;
  
  input {
    margin: 7px;
    width: 400px;
    font-size: 19px;
    background-color: #f8f7f7a8;
    height: 40px;
    border: 1px solid rgba(230, 226, 226, 0.61);
    border-radius: 3px;
    padding-left: 10px;
  }
```

---

네, 수정했습니다 !